dist/jquery.flot.js

```diff
@@ -1535,10 +1570,6 @@ Licensed under the MIT license.
                                     val = +val; // convert to number
                                     if (isNaN(val)) {
                                         val = null;
-                                    } else if (val === Infinity) {
```

Comment:
This needs to be tested. what happens if infinities reach the drawing functions ? clamt the values there ?

Maybe this part of the change dealinf with fakeInfinty should be rolled back